packages/oidc-middleware/test/e2e/specs/basic.js

```diff
@@ -22,7 +22,16 @@ describe('Basic login redirect', () => {
 
   let server;
   beforeEach(async () => {
-    server = util.createDemoServer();
+    const serverOptions = {
+      issuer: constants.ISSUER,
+      client_id: constants.CLIENT_ID,
+      client_secret: constants.CLIENT_SECRET,
+      testing: {
+          disableHttpsCheck: constants.DISABLE_HTTPS_CHECK
+        }
```

Comment:
Spacing seems to be a mix of spaces and tabs here, as the ending brace doesn't align. Mind double checking?